src/entries/popup/pages/home/Tokens.tsx

```diff
@@ -259,6 +262,7 @@ export function Tokens() {
     <Box
       width="full"
       style={{
+        height: `336px`,
```

Comment:
Seeing a small padding issue at the top of the list:
<img width="190" alt="Screenshot 2024-06-19 at 2 44 01 PM" src="https://github.com/rainbow-me/browser-extension/assets/4412473/36d20f6c-8759-44b7-bf0f-7e1bbf8b8731">